internal/pkg/mapper/branchmetadata/local_load.go

```diff
@@ -17,5 +18,22 @@ func (m *branchMetadataMapper) MapAfterLocalLoad(recipe *model.LocalLoadRecipe)
 	}
 
 	branch.Metadata = manifest.MetadataMap()
+
+	if branch.IsDefault {
+		fileToLoad := recipe.Files.
+			Load(m.state.NamingGenerator().DescriptionFilePath(".")).
+			AddMetadata(filesystem.ObjectKeyMetadata, recipe.Key()).
+			SetDescription("project description").
+			AddTag(model.FileTypeMarkdown).
+			AddTag(model.FileKindProjectDescription)
+		file, err := fileToLoad.ReadFile()
+		if err != nil {
+			return err
+		}
+		if file.Content != "" {
+			branch.Metadata[model.ProjectDescriptionMetaKey] = file.Content
```

Comment:
Tady jsem měl podle jira issue provést ještě nějakou normalizaci ale není mi jasný jakou. 🤔 Podle e2e testu to vypadá že překládá newliny ok.

Reply:
@webrouse tady teda není třeba nic víc normalizovat?

Reply:
Sorry, prehliadol som to.

Cases ktore mozu nastat:
- niekto to ulozi bez `\n` na konci (pri manualnej editacii)
- niekto bude mat na konci 3x `\n` 
- IMHO je hodnota stale rovnaka, a mohol by si aj tu zavolat tu `normalize` function, ktoru volas pri `remote load` ... 

Normalizacia je vyriesena v `ReadFileContentTo` ale to tu nevolas.

Reply:
Ahá, jasně. Ok, doplním tam ten `strings.TrimRight(str, " \r\n\t")`